packages/manager/apps/web/client/app/domain/translations/Messages_fr_FR.json

```diff
@@ -819,8 +819,10 @@
   "domain_dashboard_options_status_disabled": "Résilié",
   "domain_dashboard_options_unavailable": "Option non disponible",
   "domain_dashboard_options_desactivate_option": "Résilier",
+  "domain_dashboard_options_dns_anycast_expiry": "date d'effet: {{t0}}",
```

Comment:
```suggestion
  "domain_dashboard_options_dns_anycast_expiry": "date d'effet: {{date}}",
```
instead 😄

Reply:
This change is done. I was still thinking that we would have to name the variables as t0, t1 etc. Sorry. And thanks for pointing this out :)

Reply:
@FredericEspiau `date d'effet:` or `date d'effet :`? :)